src/algorithms.jl

```diff
@@ -665,7 +665,7 @@ end
 ImplicitEuler(;chunk_size=0,autodiff=true,diff_type=Val{:forward},
                           linsolve=DEFAULT_LINSOLVE,nlsolve=NLNewton(),
                           extrapolant=:constant,
-                          controller=:Standard) =
+                          controller=:PI) =
```

Comment:
Why PI everywhere? Is it just better now?

Reply:
Yeah, `PI` produces much better reuse.